shared_crates/ui/src/text.rs

```diff
@@ -68,3 +71,19 @@ pub fn FontAwesomeIcon(
         .to_string(),
     )
 }
+
+#[element_component]
+/// Hyperlink to a URL. This will open browser when clicked.
+pub fn Hyperlink(
+    _hooks: &mut Hooks,
+    /// The URL to open
+    url: String,
+) -> Element {
+    crate::prelude::Button::new(&url.clone(), move |world| {
+        println!("Opening url {}", &url);
+        let link = Entity::new().with(hyperlink(), url.clone());
```

Comment:
Maybe worth adding a function for this instead of an entity. That would also allow us to return an error if we find the thing that's being opened to be invalid/questionable.

Reply:
> Maybe worth adding a function for this instead of an entity. That would also allow us to return an error if we find the thing that's being opened to be invalid/questionable.

I tried before to call `webbrowser` here but get a '__wbindgen_placeholder__' error. That's why I have to send to the world and watch it. Perhaps we can do the check there and then decide whether we want to open it or not?